colmena.nix

```diff
@@ -66,4 +66,11 @@ in
   };
 
   nomad-client-cdunster = _: { };
+
+  jost-test-os-terone = _: {
+    fileSystems."/" = {
```

Comment:
I understand I no longer need this to have the machine identified. Should this be removed then?

Reply:
yes, I believe so, and maybe could you have the PR be off of https://github.com/holochain/wind-tunnel-runner/pull/9  so that they won't contend?

Reply:
That would be branch: `add-hp-nodes`

Reply:
okay